app/views/shared/_nav_auth.html.slim

```diff
@@ -0,0 +1,20 @@
+nav.bg-white
+  .container.px2.py3
+    .clearfix
+      .sm-col.mb1.sm-m0
+        = link_to \
+          image_tag(asset_url('logo.svg'),
+            alt: APP_NAME,
+            class: 'align-bottom',
+            width: 170), root_path, class: 'inline-block'
+        .mt-12p.h6.sm-show.caps
+          = link_to 'About', root_path, class: 'mr3 gray'
```

Comment:
these are just placeholders for now -- once we make some decisions on this UI element, will def add to locales